src/cli.js

```diff
@@ -43,6 +43,7 @@ const run = (paths, { ignore }) => {
     const tester = new ReactTester(Element);
     const logger = new Logger(Element, relativePath);
 
+    if (logger.validateElement(path) === false) return;
```

Comment:
This `path` isn't used, right?

Reply:
Also, let's make it `if (!logger.validateElement()) return;`

Reply:
true, was a leftover from a test before. Done :)